src/test/java/ikysil/training/dao/OrderDaoTest.java

```diff
@@ -0,0 +1,49 @@
+package ikysil.training.dao;
+
+import ikysil.training.dao.impl.InMemoryOrderDao;
+import ikysil.training.ws.api.v1.dto.OrderDto;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+public class OrderDaoTest {
```

Comment:
you testing here `InMemoryOrderDao` so test should be named `InMemoryOrderDaoTest`.
you can not test interface `OrderDao`.
This is common naming convention